Musicplayer.php

```diff
@@ -29,11 +29,11 @@
 		<div class="navigation">
 			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
-				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
+				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav_normal.php page and places them in the navigation div
-				require 'nav_normal.php'; //'require' is 100% needed for the site to run 
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
```

Comment:
26/09/2021: I create a muiscplaypage for the admin so I change the php to called up to nav_admin because in the nav admin the admin can access to some advance function like delete user